weave/trace_server/migrations/002_feedback.up.sql

```diff
@@ -0,0 +1,49 @@
+
+CREATE TABLE feedback (
+    /*
+    `id`: The unique identifier for the feedback. This is a UUID.
+    */
+    id String,
+
+    /*
+    `project_id`: The project identifier for the call. This is an internal
+    identifier that matches the project identifier in the W&B API.
+    It is stored for feedback to allow efficient permissions filtering.
+    */
+    project_id String,
+
+    /*
+    `call_id`: The unique identifier for the call. This is typically a UUID.
+    */
+    call_id String,
+
+    /*
+    `wb_user_id`: The ID of the user that created the call. This is the ID of the user in the
+    W&B API.
+    */
+    wb_user_id String NULL,
```

Comment:
Probably I should just make this non-NULL. The comment definitely needs correcting to say "feedback" instead of "call", that's just a copy+paste error.

For cases like end user feedback, the person providing the thumbs up / down might not be the same as the person/account whose credentials were used for authenticating the feedback creation transaction. So I was wavering between this column meaning "this column represents the user who provided the feedback" and "this column represents the user whose credentials were provided at the time of creation".

Reply:
I think it is probably more clear for this to be the authenticating user, and the end user providing the feedback should be an application level concern. E.g. if you embed this in some webapp, their user base is disjoint from w&b so they may want their own user_id in the json object itself.

Reply:
I agree with Jason - the structured field here should be the authenticated user adding the row to the DB. The row itself can be application-controlled to specify some internal user id if needed.

This follows the same pattern as calls. 

As far as nullability goes, the reason calls wb_user_id is nullable is just because technically "Open" projects could have rows written by unauthed users. This might be the same case for feedback (just trying to follow W&B core permissions model)

Reply:
Made non-null and added a separate "creator" column for the display name. Per Shawn we don't need to worry about supporting open projects in Weave.